tests/dashbio_demos/app_volcano_plot.py

```diff
@@ -65,213 +65,212 @@ def header_colors():
+                            children=[
+                                html.Div(
+                                    title='Number of points in the upper left',
+                                    children=[
+                                        daq.LEDDisplay(
+                                            className='vp-input-like',
+                                            label='Upper left points',
+                                            id='vp-upper-left',
+                                            size=25,
+                                            color='#19D3F3'
+                                        ),
+                                        html.Div(
+                                            className='vp-test-util-div',
+                                            id='vp-upper-left-val'
+                                        )
+                                    ]
+                                ),
+                                html.Br(),
+                                html.Div(
+                                    className='vp-vertical-style',
+                                    title='Number of points in the upper right',
+                                    children=[
+                                        daq.LEDDisplay(
+                                            className='vp-input-like',
+                                            label='Upper right points',
+                                            id='vp-upper-right',
+                                            size=25,
+                                            color='#19D3F3'
+                                        ),
+                                        html.Div(
+                                            className='vp-test-util-div',
+                                            id='vp-upper-right-val'
+                                        )
+                                    ]
+                                ),
+                            ],
+                        ),
+                        html.Hr(),
+                        html.Div(id='vp-event-data')
+                    ])
+                )
+            ])
+        ])
+    ])
 
 
 def callbacks(app):  # pylint: disable=redefined-outer-name
+    @app.callback(
+        Output('vp-event-data', 'children'),
+        [Input('vp-graph', 'hoverData'),
+         Input('vp-graph', 'clickData')]
+    )
+    def get_event_data(hover, click):
+        hover_data_div = [
+            html.Div(className='vp-option-name', children='Hover data')
+        ]
+        hover_data = 'Hover over a data point to see it here.'
+
+        if hover is not None:
+            hovered_point = hover['points'][0]
+            hover_data = [
+                'x: {}'.format(hovered_point['x']),
+                html.Br(),
+                'y: {}'.format(hovered_point['y']),
+                html.Br(),
+                hovered_point['text'].strip('<br>').replace(
```

Comment:
Is this space-padded hyphen standard gene notation?

Reply:
Not sure, but they were originally separated by a line break, which I don't think is "proper" notation -- this is just a different separator. I can replace the hyphen with putting parentheses around it (done in 7e90265)

Reply:
Yeah for sure! It's good to catch things that might look unclear. I don't have a lot of experience with bioinformatics, so I'm just as unsure as you are about these things! :P

Reply:
Ok, anyway, I personally like how it renders now, so I'll resolve... Thanks!